Webmin/input/checks/accounts_passwd_cmd.xml

```diff
@@ -0,0 +1,33 @@
+<def-group>
+  <definition class="compliance" id="accounts_passwd_cmd" version="1">
+    <metadata>
+      <title>Use passwd for changing passwords.</title>
+      <affected family="unix">
+        <product>Webmin</product>
```

Comment:
Unless this guidance works on *every* version of Webmin, you'll want to specify a version number here.